lib/logger/logger.ts

```diff
@@ -161,12 +167,45 @@ class Logger {
     const data = assign({level: logLvl}, this.globals, metadata, userdata);
     if (this.logRouter != null) {
       data._kvmeta = this.logRouter.route(data);
+    } else if (globalRouter) {
```

Comment:
above we have a `!= null` check .. should the checks on 168 and 170 be the same?